README.md

```diff
@@ -70,6 +70,7 @@ Build requirements:
  * `cmake` (v2 or above)
  * `apt install libjson-c-dev libcmocka-dev libssl-dev` *or*
  * `yum install json-c-devel libcmocka-devel openssl-devel`
+ * 'valgrind' for testing
```

Comment:
tiny nit, can you use backticks: `valgrind` to get the markup?

Reply:
Yes, thank you